src/api/subscriptions.js

```diff
@@ -317,6 +317,72 @@ export const SubscriptionFlows = {
  */
 export let Config;
 
+/**
+ * Internal params for GetEntitlements requests.
```

Comment:
What does this mean? How will it be used. More details in the comments please

Reply:
I added details to these comments. Thanks for letting me know they needed more work